src/sentry/integrations/jira/integration.py

```diff
@@ -319,6 +320,10 @@ def get_group_description(self, group, event, **kwargs):
         return "\n".join(output)
 
     def get_client(self):
+        # Allow use of mocks in local development.
+        if options.get("mocks.jira"):
```

Comment:
@mgaeta Can't we just mock the `get_client` function in tests instead of actually having to change production code like this?

Reply:
This allows you to use the mocks in local development. We do just mock `get_client` in the tests.

Reply:
@mgaeta remind me why I would want to use a mock in my local environment? Maybe I'm missing something.